{{ cookiecutter.__package_name_kebab_case }}/Dockerfile

```diff
@@ -51,7 +64,7 @@ RUN --mount=type=cache,uid=$UID,gid=$GID,target=/home/user/.cache/pypoetry/ \
 
 
 
-FROM base as ci
+FROM builder as ci
 
 # Allow CI to run as root.
 USER root
```

Comment:
I don't like these two lines. I believe we added them for compatibility with GitLab CI, but the trouble is that we're testing in an environment that differs from the environment in which we're running the app (`ci` uses the root user while `app` uses the non-root user). Do you think this is fixable in this PR, or should we address that in a separate one?

Reply:
I agree that this is a pain. And I honestly don't even remember what the exact GitLab CI bug was that we fixed with this 🙂 I'd add it to the issue list and fix it later though.

Reply:
OK, sure!